src/components/plugins/export/RemoteStoreExportControls.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { Button } from "semantic-ui-react";
+import exportUtil from "./util";
+import type {
+  RemoteStore as RemoteStoreType,
+} from "../../store";
+
+type Props = {
+  store: RemoteStoreType,
+  query: Object,
+  columnModel: Array<Object>,
+  exportButtonLabel?: string,
+  exportFileName?: string,
+}
+
+const ExportControls = (props: Props) => {
```

Comment:
I would add a brief comment to explain exactly how remote export works (e.g. filtering is applied but pagination is not).